README.md

```diff
@@ -7,7 +7,7 @@ Many thanks to and inspiration from: https://github.com/arankwende/meshtastic-mq
 
 Requires packages: Meshtastic, Paho-MQTT v2, Tkinter, cryptography which might be installed with:
 
-`pip3 install meshtastic paho-mqtt tk cryptography`
+`pip3 install meshtastic paho-mqtt tkinter cryptography`
```

Comment:
It's been and while and I'm not sure this is the correct way to install tkinter.  I'm finding mixed info on this.

Reply:
Ah thanks. In practice I didn’t use pip to install any of them. I just noticed that [`tk`](https://pypi.org/project/tk/) is something else entirely.

Looks like `tkinter` isn’t in PyPI though so this clearly isn’t right.

Reply:
Yeah, not sure what's going on.  I'm on a mac which uses brew to install python-tk, so maybe I was just confused when I wrote this.